lms/static/sass/discussion/views/_inline.scss

```diff
@@ -35,7 +35,6 @@
 
     .wmd-preview-container {
         @include discussion-new-post-wmd-preview-container;
-        margin-bottom: $baseline;
```

Comment:
I realize now it is different in teams... Ok to ignore this for now and put into separate ticket if preferred. 
![image](https://cloud.githubusercontent.com/assets/2023680/21165168/b4c2d656-c16b-11e6-9fee-e8f551ce44ee.png)